boards/arm/v2m_musca/doc/v2m_musca.rst

```diff
@@ -0,0 +1,412 @@
+- A physical UART connection which is relayed over interface USB Serial port.
+
+This interfaces are exposed via DAPLink which provides:
+
+- Serial Wire Debug (SWD).
+- USB Mass Storage Device (USBMSD).
+- UART.
+- Remote reset.
+
+For more details please refer
+to `DAPLink Website`_.
+
+
+Building a secure only application
+----------------------------------
+
+
+You can build applications in the usual way. Here is an example for
+the :ref:`hello_world` application.
+
+
+Building a secure/non-secure with Zephyr on both sides
+------------------------------------------------------
+
+The process is similar to before, with the added configuration of setting-up
+the secure/non-secure configuration entry on zephyr.
+
+
+Building a secure/non-secure with Trusted Firmware
+--------------------------------------------------
+
+The process requires five steps:
+
+1. Build Trusted Firmware (tfm).
+2. Import it as a library to the Zepyhr source folder.
+3. Build Zephyr with a non-secure configuration.
+4. Merge the two binaries together and sign them.
+5. Concatenate the bootloader with the signed image blob.
+
+In order to build tfm please refer to `Trusted Firmware M Guide`_.
+Follow the build steps for AN521 target while replacing the platform with
+``-DTARGET_PLATFORM=MUSCA_A`` and compiler (if required) with ``-DCOMPILER=GCC``
+
+Copy over tf-m as a library to the zephyr project source and create a shortcut
+for the secure veneers.
+
+.. code-block:: bash
+
+   cp -r install/ $ZEPHYR_PROJECT/src/ext
+   cp $ZEPHYR_PROJECT/src/ext/install/export/tfm/veneers/s_veneers.o $ZEPHYR_PROJECT/src/ext
+
+
+Build the Zephyr app in the usual way.
+
+Uploading an application to V2M Musca
+-------------------------------------
+
+In order to flash an application to V2M Musca, it needs to be converted to
+Intel's hex format. An optional bootloader can be prepended to the image.
+The QSPI flash base address alias is 0x200000.
+
+The image off-set is calculated by adding the flash off-set to the
+bootloader partition size.
+
+A third-party tool (srecord) is used to generate the Intel formatted hex image. For more information
+refer to the `Srecord Manual`_.
+
+.. code-block:: bash
+
+   srec_cat $BIN_BOOLOADER -Binary -offset $QSPI_FLASH_OFFSET $BIN_SNS -Binary -offset $IMAGE_OFFSET -o $HEX_FLASHABLE -Intel
+
+   # For a 64K bootloader IMAGE_OFFSET = $QSPI_FLASH_OFFSET + 0x10000
+   srec_cat $BIN_BOOLOADER -Binary -offset 0x200000 $BIN_SNS -Binary -offset 0x210000 -o $HEX_FLASHABLE -Intel
+
+   # For a 256K bootloader IMAGE_OFFSET = $QSPI_FLASH_OFFSET + 0x40000
+   srec_cat $BIN_BOOLOADER -Binary -offset 0x200000 $BIN_SNS -Binary -offset 0x240000 -o $HEX_FLASHABLE -Intel
+
+Connect the V2M Musca to your host computer using the USB port and you should
+see a USB connection which exposes a Mass Storage (MBED) and a USB Serial Port.
+Copy the generated zephyr.hex in the MBED drive.
+Reset the board and you should be able to see on the corresponding Serial Port
```

Comment:
Change to:

    Reset the board, and you should see the following message on the corresponding
    serial port: